site/third-party-projects.html

```diff
@@ -220,25 +226,17 @@ <h2>Infrastructure Projects</h2>
 running Spark</li>
   <li><a href="http://alluxio.org/">Alluxio</a> (née Tachyon) - Memory speed virtual distributed 
 storage system that supports running Spark</li>
-  <li><a href="https://github.com/datastax/spark-cassandra-connector">Spark Cassandra Connector</a> - 
-Easily load your Cassandra data into Spark and Spark SQL; from Datastax</li>
   <li><a href="https://github.com/filodb/FiloDB">FiloDB</a> - a Spark integrated analytical/columnar 
 database, with in-memory option capable of sub-second concurrent queries</li>
   <li><a href="http://www.elasticsearch.org/guide/en/elasticsearch/hadoop/master/spark.html#spark-sql">ElasticSearch - 
 Spark SQL</a> Integration</li>
-  <li><a href="https://github.com/tresata/spark-scalding">Spark-Scalding</a> - Easily transition 
-Cascading/Scalding code to Spark</li>
   <li><a href="http://zeppelin-project.org/">Zeppelin</a> - an IPython-like notebook for Spark. There 
 is also <a href="https://github.com/tribbloid/ISpark">ISpark</a>, and the 
```

Comment:
Spark seems unmaintained so I'd remove it.

Reply:
Spark Notebook is also problematic w.r.t the trademark policy, so we should reach out to it as well.

Reply:
I'd not call Zeppelin `an IPython-like notebook for Spark` either.